source/jormungandr/tests/routing_tests_new_default.py

```diff
@@ -327,6 +327,12 @@ def test_same_origin_and_destination_point_in_request(self):
         assert response[1] == 400
         assert "your origin and destination points are the same" in response[0]["message"]
 
+    def test_walking_transfer_penalty_parameter(self):
+        """
+        This feature is no longer supported
+        """
+        pass
+
```

Comment:
Why do you add this test ?

Reply:
As i explained above new_default doesn't work and will certainly disappear in future.

Reply:
but you have to add it to `routing_tests_new_default.py` ?

Reply:
Yes